wiki/SourceCode.md

```diff
@@ -83,7 +83,7 @@ rights).
 
 This is normally given on a case by case basis, and the best place to
 discuss getting write access is on the [Biopython mailing
-list](mailto:biopython@biopython.org).
+lists](https://biopython.org/wiki/Mailing_lists).
```

Comment:
You can give the relative URL here, just ``[Biopython mailing list](Mailing_lists)`` should work. This is subtle but it would mean the link keeps working if we ever reorganise the pages (e.g. to remove the word ``wiki`` from the URL since this is no longer in a wiki).